A51-custom-backend-metrics.md

```diff
@@ -78,27 +77,36 @@ function recordRequestCostMetric(String name, double value);
 // Records a utilization metric measurement for the call.
 function recordUtilizationMetric(String name, double value);
 
+// Records an opaque named metric measurement for the call.
+function recordNamedMetricsMetric(String name, double value);
```

Comment:
The named_metrics and eps changes do not belong in this PR.  We'll make those changes in a separate PR.

Reply:
Removed. I think I accidentally rebased on top of the follow-up change (eps/named metrics).